website/docs/r/vapp_network.html.markdown

```diff
@@ -57,8 +92,15 @@ The following arguments are supported:
 * `name` - (Required) A unique name for the network.
 * `description` - (Optional; *v2.7+*, *vCD 9.5+*) Description of vApp network
 * `vapp_name` - (Required) The vApp this network belongs to.
-* `netmask` - (Optional) The netmask for the new network. Default is `255.255.255.0`.
+* `netmask` - (Deprecated) Use `prefix_length` instead. The netmask for the new network.
+
+~> **Warning:** In `v3.9.0`, field `netmask` no longer has a `default` value of  `255.255.255.0` so that IPv6 can be supported using the new `prefix_length` field. 
+This change makes `terraform validate|plan` fail if the user didn't provide a value earlier and relied on default `255.255.255.0`.
+In case that happens, a user needs to add `"netmask" = "255.255.255.0"` to existing vApp networks.
+* `prefix_length` - (Optional) The subnet prefix length for the network.
 * `gateway` - (Required) The gateway for this network.
+~> **Note:** VCD returns IPv6 addresses in extended-shortened format e.g `fe80:0:a:ab:0:abc:abcd:aaaa`, it is up to the user
```

Comment:
For some reason this note doesn't render:

![Screenshot 2023-03-16 at 17 05 44](https://user-images.githubusercontent.com/11330689/225659646-309a8af5-6b5f-4a33-ac7b-75d1904216f4.png)

Reply:
Fixed.